src/content/en/updates/posts/2016/04/devtools-digest-command-menu.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: updates/post
+title: "DevTools Digest: More power with the new Command Menu"
+published_on: 2016-04-22
+updated_on: 2016-04-22
+authors:
+  - pbakaus
+tags:
+  - devtools
+  - digest
+  - update
+description: "Read about DevTools' new command menu and its over 60 actions that enable crazy fast workflows."
+featured_image: /web/updates/images/2016/04/devtools-digest/command-menu.png
+---
+
+<p class="intro">Read about DevTools' new <strong>command menu</strong> and its over 60 actions that enable crazy fast workflows.</p>
+
+## CMD+Shift+P to bring up the Command Menu
+
+![Command Menu in DevTools](/web/updates/images/2016/04/devtools-digest/command-menu.png)
+
+The "Jump to File" dialog that appears when you press <kbd>CMD+P</kbd> in the Sources panel isn't terribly well known, but has been around for a while. We've now went much further than that and developed a text-editor-inspired command menu that can drive almost every important action in DevTools.
+
+Hit <kbd>CMD+Shift+P</kbd> anywhere (even when the page is focussed!) to bring up the Command Menu, then type to filter and hit return to trigger the action. A few sample actions you could try:
+
+  * Appearance: Switch to Dark Theme
+  * DevTools: Dock to bottom
+  * Mobile: Inspect Devices...
+  * Network: Go offline
+
+The new command menu is a super quick way to navigate and discover new settings and actions across DevTools.
+
+## Pretty print HTML
+
+![Pretty HTML](/web/updates/images/2016/04/devtools-digest/pretty-html.jpg)
+
+We've had prenty print built into Sources for a while for JS and CSS sources, but have just extended it to full blown HTML pretty printing. Give it a try – it doesn't just reformat HTML, but also the JavaScript and Styles within it!
+
+- - -
+
+As always, [let us know what you think via
+Twitter](https://twitter.com/intent/tweet?text=%40ChromeDevTools) or the
+comments below, and submit bugs to [crbug.com/new](https://crbug.com/new).
+
+Until next month!
+Paul Bakaus & the DevTools team
```

Comment:
I may be missing something, but it looks like console autocompletion has a referenced screenshot but isn't summarized in this digest. Intentional? :)